test/hotspot/jtreg/runtime/appcds/MoveJDKTest.java

```diff
@@ -42,19 +40,21 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.nio.file.StandardCopyOption;
+import jdk.test.lib.cds.CDSTestUtils;
 import jdk.test.lib.process.OutputAnalyzer;
 
 public class MoveJDKTest {
     public static void main(String[] args) throws Exception {
         String java_home_src = System.getProperty("java.home");
-        String java_home_dst = System.getProperty("user.dir") + File.separator + "moved_jdk";
+        String java_home_dst = CDSTestUtils.getOutputDir() + File.separator + "moved_jdk";
 
```

Comment:
Includes the changes from [JDK-8251267](https://bugs.openjdk.org/browse/JDK-8251267) ( commit [291ba97](https://github.com/openjdk/jdk/commit/291ba97fab29bdd1e423aff8e660d9d6311d7c89) ) on this file